src/app/components/api-key/api-key-registration.jsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import {makeStyles} from '@material-ui/core/styles';
+import {getAngularService} from './';
+import { ThemeProvider } from '@material-ui/core/styles';
+import theme from '../../themes/theme';
+import Button from '@material-ui/core/Button';
+import Card from '@material-ui/core/Card';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import CardHeader from '@material-ui/core/CardHeader';
+import TextField from '@material-ui/core/TextField';
+import Typography from '@material-ui/core/Typography';
+import { useFormik } from 'formik';
+import * as yup from 'yup';
+
+const useStyles = makeStyles(() => ({
+  grid: {
+    display: 'grid',
+    gridTemplateColumns: '1fr',
+    gridRowGap: '8px',
+  },
+}));
+
+const validationSchema = yup.object({
+  email: yup.string()
+    .required('Email is required')
+    .email('Enter a valid email'),
+  nameOrganization: yup.string()
+    .required('Name or Organization is required'),
+});
+
+function ChplApiKeyRegistration () {
+  //const $log = getAngularService('$log');
+  const networkService = getAngularService('networkService');
+  const toaster = getAngularService('toaster');
+
+  const formik = useFormik({
+    initialValues: {email: '', nameOrganization: ''},
+    validationSchema: validationSchema,
+    onSubmit: (values) => {
+      createRequest(values);
+    },
+    validateOnChange: false,
+    validateOnMount: true,
+  });
+
+  const createRequest = (values) => {
```

Comment:
I usually don't include the parentheses when the function has only one parameter. I'm not against having it, but if we choose to go that way we should make that part of our style guide, whether or not our linting tools can catch it

```suggestion
  const createRequest = values => {
```